### PR TITLE
Feature/MOS-7 Show Workshop's Detail

### PR DESCRIPTION
**Pull request by chrsomle**
# Description
Redirect to maps when user click distance component.

# Ticket
https://berkatjayagroup.atlassian.net/browse/MOS-112

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: Moku/Customer/Components/CollectionInfoDetailBengkel.swift [hunk 1]
```diff
@@ -14,16 +14,38 @@ struct CollectionInfoDetailBengkel: View {
...
-                Text("\(mainInfo)")
-                    .fontWeight(.bold)
+                Text("\(mainInfo)").font(.headline)
+            }
+            Button {
+                onTap?()
+            } label: {
+                Text(cta).font(.caption2.bold())
+            }
+            if cta == "Lihat Detail"{
```
Comment (zeinrezky) [inline Moku/Customer/Components/CollectionInfoDetailBengkel.swift]: put 'Lihat Detail' as enum instead

File: Moku/Customer/Views/BengkelDetail.swift [hunk 2]
```diff
@@ -67,7 +67,9 @@ struct BengkelDetail: View {
                             .frame(width: proxy.size.width * 0.3, alignment: .center)
                             .background(Color(hex: "F3F3F3"))
                             .cornerRadius(8)
-                        CollectionInfoDetailBengkel(titleInfo: "Jarak dari Anda", imageInfo: "", mainInfo: viewModel.distance, cta: "Lihat Peta")
+                        CollectionInfoDetailBengkel(titleInfo: "Jarak dari Anda", imageInfo: "", mainInfo: viewModel.distance, cta: "Lihat Peta") {
+                            MapHelper.direct(bengkel: viewModel.bengkel)
+                        }
                             .padding(.all, 4)
                             .frame(width: proxy.size.width * 0.3, alignment: .center)
                             .background(Color(hex: "F3F3F3"))
```
Comment (zeinrezky) [inline Moku/Customer/Views/BengkelDetail.swift]: put this hex in constant instead

File: Moku/Helpers/MapHelper.swift [hunk 3]
```diff
@@ -89,6 +88,33 @@ struct MapHelper {
 
         return "N/A"
     }
+
+    static func direct(bengkel: Bengkel) {
+        let gmapAvailable = UIApplication.shared.canOpenURL(URL(string: "comgooglemaps://")!)
```
Comment (zeinrezky) [inline Moku/Helpers/MapHelper.swift:93]: Put all the function contains ".shared" in available Os mode, since this will no longer used in 15.

**Review by zeinrezky — CHANGES_REQUESTED** [review]

File: Moku/Helpers/MapHelper.swift (hunk 3 above)
Comment (chrsomle) [reply Moku/Helpers/MapHelper.swift:93]: @zeinrezky ada artikel for related matters ga kak? drtd kyknya nemu yang deprecated di iOS 15 cuma `UIApplication.shared.windows` kak

**Pull request merged**